app/views/batch_processes/show_parent.html.erb

```diff
@@ -0,0 +1,91 @@
+<h1>Parent Batch Process Detail</h1>
+<p id="notice"><%= notice %></p>
+<p>
+  <strong>Batch Process ID</strong>
+  <%= link_to @batch_process.id, batch_process_path(@batch_process) %>
+</p>
+<p>
+  <strong>Parent Object</strong>
+  <%= if @parent_object.class == ParentObject
+                  link_to "#{@parent_object.id} (current record)", parent_object_path(@parent_object)
+      else
+        "#{@parent_object} (pending or deleted)"
+      end
+  %>
+</p>
+<p>
+  <strong>Status</strong>
+  <%= "#{@parent_object.status_for_batch_process(@batch_process.id)}" %>
+</p>
+<p>
+  <strong>Duration</strong>
+  <%=  "#{@parent_object.duration_for_batch_process(@batch_process.id)}" %>
+</p>
+
+<table class="table table-bordered table-striped">
+  <thead class="thead-dark">
+    <tr>
+      <th scope="col"> Ingest step </th>
+      <th scope="col"> Time </th>
+    </tr>
+  </thead>
+  <tr>
+    <td>Submitted</td>
+    <td class="submission_time"><%= @batch_process&.created_at %></td>
+  </tr>
+  <tr>
+    <td>Processing Queued</td>
+    <td class="processing_queued_time"><%= @notes["processing-queued"] if @notes %></td>
+  </tr>
+  <tr>
+    <td>Metadata Fetched</td>
+    <td class="metadata_fetched"><%= @notes["metadata-fetched"] if @notes %></td>
+  </tr>
+  <tr>
+    <td>Child Records Created</td>
+    <td class="children_created"><%= @notes["child-records-created"] if @notes %></td>
+  </tr>
+  <tr>
+    <td>Manifest Saved</td>
+    <td class="manifest_saved"><%= @notes["manifest-saved"] if @notes %></td>
+  </tr>
+  <tr>
+    <td>Solr Indexed</td>
+    <td class="solr_indexed"><%= @notes["solr-indexed"] if @notes %></td>
+  </tr>
+</table>
+<table class="table table-bordered table-striped">
+  <thead class="thead-dark">
+    <tr>
+      <th scope="col"> Child object oid </th>
+      <th scope="col"> Last ptiff conversion </th>
+    </tr>
+  </thead>
+  <% if @parent_object&.class == ParentObject %>
```

Comment:
shouldn't this be a data table like the others?

Reply:
I agree with you @alishaevn, though for the sake of getting this in, I think we break that out to a separate PR